app/settings.py

```diff
@@ -99,3 +99,5 @@ def get_key(_key_name):
 
 # JWT configurations
 EQ_JWT_LEEWAY_IN_SECONDS = 120
+
+EQ_ROLE_PERMISSIONS = {'admin': ['flush']}
```

Comment:
I wonder if we need a nested structure with permissions at this point, would a flusher role suffice?